src/shared/glemu.h

```diff
@@ -40,23 +40,19 @@ namespace gle
     GLE_DEFATTRIB(boneweight, ATTRIB_BONEWEIGHT, 4, GL_UNSIGNED_BYTE)
     GLE_DEFATTRIB(boneindex, ATTRIB_BONEINDEX, 4, GL_UNSIGNED_BYTE)
 
-    #define GLE_INITATTRIB(name, index, suffix, type) \
-        static inline void name##suffix(type x) { glVertexAttrib1##suffix##_(index, x); } \
-        static inline void name##suffix(type x, type y) { glVertexAttrib2##suffix##_(index, x, y); } \
-        static inline void name##suffix(type x, type y, type z) { glVertexAttrib3##suffix##_(index, x, y, z); } \
-        static inline void name##suffix(type x, type y, type z, type w) { glVertexAttrib4##suffix##_(index, x, y, z, w); }
     #define GLE_INITATTRIBF(name, index) \
-        GLE_INITATTRIB(name, index, f, float) \
+        static inline void name##f(float x) { glVertexAttrib1##f##_(index, x); } \
+        static inline void name##f(float x, float y) { glVertexAttrib2##f##_(index, x, y); } \
+        static inline void name##f(float x, float y, float z) { glVertexAttrib3##f##_(index, x, y, z); } \
+        static inline void name##f(float x, float y, float z, float w) { glVertexAttrib4##f##_(index, x, y, z, w); } \
```

Comment:
`f` is not a macro parameter anymore so it doesn't need to be glued using `##`.

```suggestion
        static inline void name##f(float x, float y, float z, float w) { glVertexAttrib4f_(index, x, y, z, w); } \
```

Reply:
Nice catch, thanks!